src/components/media/MediaBody.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import classnames from 'classnames';
+
+import './Media.scss';
+
+export interface MediaBodyProps {
+    /** Child elements */
+    children: React.ReactNode;
+
+    /** Additional class names */
+    className?: string;
+
+    /** Additional inline styles */
+    style?: React.CSSProperties;
+}
+
+const MediaBody = ({ className, children, ...rest }: MediaBodyProps) => (
```

Comment:
Will the MediaBodyProps interface work with ...rest props like `data-testid`, `aria-label`, etc.?

Reply:
I believe that's part of the `JSX.IntrinsicAttributes` interface?